Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -475,15 +489,33 @@ void Renderer::UpdateDrawRectangle(int backbuffer_width, int backbuffer_height)
 	}
 
 	// Check for force-settings and override.
-	if (g_ActiveConfig.iAspectRatio == ASPECT_FORCE_16_9)
+	switch (g_ActiveConfig.iAspectRatio)
+		{
+	case ASPECT_FORCE_16_9:
 		use16_9 = true;
-	else if (g_ActiveConfig.iAspectRatio == ASPECT_FORCE_4_3)
+		break;
+	case ASPECT_ANALOG_WIDE:
+		use16_9 = true;
+		break;
+	case ASPECT_FORCE_4_3:
 		use16_9 = false;
-
+		break;
+	case ASPECT_ANALOG:
+		use16_9 = false;
+		break;
+	default:
+		use16_9 = g_aspect_wide;
+		break;
+	}
 	if (g_ActiveConfig.iAspectRatio != ASPECT_STRETCH)
 	{
 		// The rendering window aspect ratio as a proportion of the 4:3 or 16:9 ratio
 		float Ratio = (WinWidth / WinHeight) / (!use16_9 ? (4.0f / 3.0f) : (16.0f / 9.0f));
+		if (g_ActiveConfig.iAspectRatio == ASPECT_ANALOG || g_ActiveConfig.iAspectRatio == ASPECT_ANALOG_WIDE)
+		{
+			Ratio = (WinWidth / WinHeight) / VideoInterface::GetAspectRatio(use16_9);
```

Comment:
I'd inline the "Ratio" variable into the switch() above. So there won't be a need for use16_9 any more.